core/common/src/thrift/file_system_master.thrift

```diff
@@ -25,8 +25,16 @@ struct MountTOptions {
   2: optional map<string, string> properties
 }
 
+enum LoadMetadataTypeT {
```

Comment:
I don't know. Is the convention to put T before the last word? That sounds werid. But I can change it if you prefer.